Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs

```diff
@@ -55,29 +60,28 @@ public void CreateGroupAndDeleteGroupTest()
 
 		public void CleanUpTestGroups()
 		{
-			rsapiClient.APIOptions.WorkspaceID = -1;
-			Condition groupQueryCondition = new TextCondition(GroupFieldNames.Name, TextConditionEnum.EqualTo, groupName);
-			Query<kCura.Relativity.Client.DTOs.Group> groupQuery = new Query<kCura.Relativity.Client.DTOs.Group>(FieldValue.AllFields, groupQueryCondition, new List<Sort>());
 			try
 			{
-				QueryResultSet<kCura.Relativity.Client.DTOs.Group> queryResultSet = rsapiClient.Repositories.Group.Query(groupQuery);
-				if (!queryResultSet.Success)
-				{
-					throw new Exception("Failed to Query for Test Groups");
-				}
-
-				if (queryResultSet.Results.Count > 0)
+				using (IObjectManager objectManager = serviceFactory.CreateProxy<IObjectManager>())
 				{
-					List<int> groupArtifactIds = new List<int>();
-					foreach (Result<kCura.Relativity.Client.DTOs.Group> groupResult in queryResultSet.Results)
+					QueryRequest clientQueryRequest = new QueryRequest
 					{
-						groupArtifactIds.Add(groupResult.Artifact.ArtifactID);
-					}
-
-					WriteResultSet<kCura.Relativity.Client.DTOs.Group> deleteResultSet = rsapiClient.Repositories.Group.Delete(groupArtifactIds);
-					if (!deleteResultSet.Success)
+						ObjectType = new ObjectTypeRef
+						{
+							ArtifactTypeID = 3
+						},
+						Condition = $"'Name' == '{groupName}'"
+					};
+					Services.Objects.DataContracts.QueryResult queryResult = objectManager.QueryAsync(-1, clientQueryRequest, 0, 10).GetAwaiter().GetResult();
```

Comment:
According to Thor's comments on a PR of mine, we should be setting .ConfigureAwait(false) when not using async operations. here's an example: https://github.com/relativitydev/relativity-test-helpers/blob/develop/Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldHelper.cs#L61

Reply:
Done